queries/usd/indents.scm

```diff
@@ -0,0 +1,19 @@
+[
+  (block)
+  (dictionary)
+  (list)
+  (list_proxy)
+  (metadata)
+  (timeSamples)
+  (tuple)
+  (variant_set_definition)
```

Comment:
Can you add a comment at EOL for a preview of what it is? i.e. `(block) ; { ... }`

Reply:
No problem. I'm assuming by EOL you mean an inline comment. I tried to summarize each ambiguous named node, please let me know if that's what you had in mind when you get the chance